kubernetes/loculus/values.schema.json

```diff
@@ -798,6 +798,14 @@
       "type": ["string", "null"],
       "description": "A custom welcome message to be shown on the landing page"
     },
+    "groupCreationGuidelines": {
```

Comment:
Would it make sense to have this again as an HTML as we do for a few other fields or do you think that it's better to have an array of non-HTML strings here?